Source/NVActivityIndicatorView/NVActivityIndicatorView.swift

```diff
@@ -64,7 +64,7 @@ import UIKit
  - AudioEqualizer:          AudioEqualizer animation.
  - CircleStrokeSpin:        CircleStrokeSpin animation.
  */
-public enum NVActivityIndicatorType: Int {
+public enum NVActivityIndicatorType: Int, CaseIterable {
```

Comment:
We can get rid of `Int` inheritance as well since it was introduced to implement `allTypes`.

Reply:
Thanks. I removed it.